pkg/config/config_template.yaml

```diff
@@ -2142,6 +2142,16 @@ api_key:
 #
 # jmx_use_container_support: false
 
+## @param jmx_max_ram_percentage - float - optional - default: 25.0
+## @env DD_JMX_MAX_RAM_PERCENTAGE - float - optional - default: 25.0
+## When running in a container with jmx_use_container_support enabled, the JVM can
```

Comment:
This does a good job documenting what happens when this is used with `UseContainerSupport`, what happens when its used and `jmx_use_container_support` is _false_?